src/components/topbar/profile/profile.tsx

```diff
@@ -42,21 +42,23 @@ const Profile: FC = () => {
 
   return (
     <>
-      <img src={elon} alt="" className="profilepic" onClick={showOwner} />
-
+      {owned.isJust() && (
```

Comment:
Are we sure that we want to use this operator? Or do we only want to render the `img` if `owned.isJust()` is true? If so, we'd probably want something like `owned.isJust()? <img/> : null` instead.

Reply:
done